.cdsrc.json

```diff
@@ -0,0 +1,5 @@
+{
+	"sql": {
+		"dialect": "plain"
```

Comment:
With the latest @sap/cds-dk versions this configuration is no longer required. As `cds compile -2 sql` in contrast to `cds compile -2 sqlite` already implies dialect plain!

Reply:
oh okay :( Yes I think this is only possible as of `@sap/cds@5.3.0`, but you seem to use `@sap/cds@5.1.4` in `package-lock.json`

Reply:
@stewsk any insight on that? we'd like to build plain SQL for H2 without adding further configuration to our CDS call (or via cdsrc.json). Is that possible with the currently configured CDS version?